src/server/api/redirect.ts

```diff
@@ -171,18 +180,52 @@ export default async function redirect(req: Express.Request, res: Express.Respon
     if (gaTrackingId) gaLogging(req, res, shortUrl, longUrl)
 
     // Redirect immediately if a crawler is visiting the site
-    if (isCrawler(req.headers["user-agent"] || '')) {
+    if (isCrawler(req.headers['user-agent'] || '')) {
```

Comment:
Apparently this is the canonical way to obtain the header according to [Express docs](https://expressjs.com/en/api.html)
```suggestion
    if (isCrawler(req.get('user-agent') || '')) {
```